chain/epoch_manager/src/lib.rs

```diff
@@ -77,6 +77,8 @@ impl EpochManager {
                 validator_reward,
                 0,
             )?;
+            // HACK: dummy block info. This dummy is stored instead of genesis block at height = 0.
+            // Unclear how it works, but hard to change at this point.
```

Comment:
why is it unclear how it works? Genesis block is treated specially anyways.

Reply:
- dummy block is a parent of genesis block, but has the same height
- dummy block is its own parent
- dummy block and genesis block are both in epoch 0
- epoch 0 and epoch 1 have the same epoch id

very hard to reason about correctness of functions that don't necessarily have `if (close_to_genesis(block)) { special_treatment }`.

It seems like we can view this situation as dummy block being the start of the chain with `height = -1` and the height field is not accessed by anyone, but I'm not sure.